src/services/expenses.js

```diff
@@ -0,0 +1,49 @@
+'use strict';
+
+const { v4: uuidv4 } = require('uuid');
+
+let expenses = [];
+
+function getAll(querry) {
+  return expenses;
+};
+
+function getById(expensId) {
+  const foundExpens = expenses.find(expens => expens.id === expensId);
```

Comment:
Change `foundExpens` to `foundExpense` for better readability and consistency with the task description.